mtags/src/main/scala-3/scala/meta/internal/pc/MetalsInteractive.scala

```diff
@@ -345,6 +345,26 @@ object MetalsInteractive:
     }
   end ApplySelect
 
+  object ExtensionMethodCallSymbol:
+    def unapply(tree: Tree)(using Context): Option[(Ident | Select)] =
```

Comment:
```suggestion
    def unapply(tree: Tree)(using Context): Option[Symbol] =
```
Could this return a symbol directly?

Reply:
I use the tree in `PcCollector`.

Reply:
Ah right, thanks, I missed it.